performance-tools/performance-cluster/scripts/helpers/create-cluster.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
A more generic rewrite of the `create-cluster.sh` script that already exists, would probably avoid the generation of duplicate code here.